src/tests/nativeaot/SmokeTests/Exceptions/GenericExceptions.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+public class GenericExceptions
```

Comment:
Should we instead remove these exclusions https://github.com/dotnet/runtime/blob/f0e7a686df57fc68fe3de496f9e78a55679987aa/src/tests/issues.targets#L1375-L1391

Reply:
Thanks! I assume having both would not hurt :)